### DISCUSSION TITLE
No backlink from Actions detail screen to pull request

### DISCUSSION THREAD

**Discussion by danez** [community/community#40072] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be really nice to have a link back to the related PR when looking at the Github Actions log UI . The PR number is already there in the title but no link anywhere to be found to navigate back to the PR (Screenshot 1).

There is a link on the Summary view (Screenshot 2) but it would be nice to have one on the details view as well.

Maybe could the Pr number be made a link in the title?

Screenshot 1

<img width="2457" alt="Screenshot 2022-11-25 at 11 49 00" src="https://user-images.githubusercontent.com/231804/203967929-6d95cec2-3066-494b-83dd-1f022acf6d41.png">

Screenshot 2

<img width="283" alt="Screenshot 2022-11-25 at 11 53 56" src="https://user-images.githubusercontent.com/231804/203968405-f7ffff44-8e7b-48fe-8502-33e9166495b7.png">

**Comment by Akash1134** [comment]
Thanks for sharing the feedback @danez I have added relevant labels to the discussion so our product team can review your request.
